tests/data/experiments/dummyregressor-custom-objective/output/DummyRegressor_custom_objective_eval_short.csv

```diff
@@ -1,2 +1,2 @@
 ,N,R2.raw_trim,RMSE.raw_trim,SMD.raw_trim,SMD.raw_trim_round,adj_agr.raw_trim_round,corr.raw_trim,exact_agr.raw_trim_round,h_mean,h_sd,kappa.raw_trim_round,sys_mean.raw_trim,sys_mean.raw_trim_round,sys_sd.raw_trim,sys_sd.raw_trim_round,wtkappa.raw_trim_round
```

Comment:
Given that we now have new kappa computation we need to switch this table from rounded *wtkappa* to raw wtkappa.